http/http_service.go

```diff
@@ -264,7 +262,7 @@ func (httpSvc *HttpService) unlockHandler(c echo.Context) error {
 		Event: "nwc_unlocked",
 	})
 
-	return c.NoContent(http.StatusNoContent)
+	return c.JSON(http.StatusOK, token)
```

Comment:
what JSON response is this?
If I understand this correctly this is just a string? I am wondering if some consistent JSON structure is better?

Reply:
sounds good. It can be an object with a token string field

Reply:
updated